### ISSUE TITLE
Sad news about Dijji

### ISSUE THREAD

**Issue by seeemilyplay** [Dijji/XstReader#51]
Hi there,

I'm very sad to inform you that Dijji, my dad, died at the end of May 2021. He loved working on his open source projects, so I very much hope that you have found some value in them.

If you have the spare money, and feel you would like to pay your respects in some way, then please donate to the MS Society (or the equivalent in your home country) https://www.mssociety.org.uk/

If you feel that you would like to take ownership of one of his repos please let me know. I'm a computer programmer too (he taught me when I was young), but C# and Windows aren't really something I know much about.

All the best,
Emily

**Comment by halueda** [comment]
I'm so surprised to hear about the sudden passing of your father.
Dijji helped me to read an old XST file by providing XstReader, which is only one OSS to read it.

I'm not so familiar with C# that I cannot play the role of the ownership of XstReader, now. 
But, I'm still interested in it, and, I may contribute to the project in the future.
So, please don't close the project and leave options to someone, including me, to take over.

My deepest condolences.

**Comment by iluvadev** [comment]
I am shocked by the sad news. I'm so sorry for your loss.

I am a back-end computer programmer in c#. I think this project is very useful and can grow. I have been involved in the development of some branches. If no one else wants to take ownership, I can manage this repo, acknowledging Dijji's great work, of course.

My deepest condolences.

**Comment by seeemilyplay** [comment]
Thank you for your kind words @halueda. @Dijji would've been so pleased to hear that he helped you. I definitely wouldn't close the project. I actually don't even have rights to do it anyway - I need to apply to Github for that.

**Comment by seeemilyplay** [comment]
Thank you @iluvadev. I wasn't expecting a volunteer so quickly ! I see you've got a fork going already, do you think it might be enough to redirect people to the fork?

I'll send an email to Github support to see what they can do.

**Comment by iluvadev** [comment]
Yes, I think redirecting people to the fork would be a good and easy solution.

**Comment by gitrkenn** [comment]
> Hi there,
> 
> I'm very sad to inform you that Dijji, my dad, died at the end of May 2021. He loved working on his open source projects, so I very much hope that you have found some value in them.
> 
> If you have the spare money, and feel you would like to pay your respects in some way, then please donate to the MS Society (or the equivalent in your home country) https://www.mssociety.org.uk/
> 
> If you feel that you would like to take ownership of one of his repos please let me know. I'm a computer programmer too (he taught me when I was young), but C# and Windows aren't really something I know much about.
> 
> All the best, Emily

Dear seeemilyplay,

I had interacted with Dijjiin around Nov. 2020 and worked with him briefly to resolve a minor bug.  Just today, I was revisiting this repository in the attempt to contact Dijji on a separate matter in collaboration when I read about the loss of your Dad.  Please accept my sincerest and late condolences.  Dijji's work is one of the few, if not the only one, that does much more and has so much potentials, and that was what I wanted to discuss with Dijji about due to recent needs on my end for deep diving into further development dealing with Outlook's PST format.

Anyway, if I can be of any assistance with this repository, please let me know, although I would find it difficult to fill Dijji's shoes in taking ownership of this repository.

**Comment by paulpv** [comment]
@seeemilyplay I am new here, but sorry to hear about your father. I did not know @Dijji , but hope all has gone as well as it can.

I have 3 young girls (9, 7, & 5), and I am a 51 year old programmer (lots of C#/C++/C/Qt/Java/Kotlin/Python/Swift/...) with involvement/ownership of more than a few open source projects, so your post here struck a small nerve.

Your post was awhile ago, and it looks like there are plenty of forks, but if you were ever able to get ownership of this repo and you are still looking for a maintainer, I could consider adding this to my plate.

My main project that I work on is https://github.com/DistroAV/DistroAV (Qt/C++ based).  
I have plenty of other projects in my personal and "alt" GitHub accounts:
* https://github.com/paulpv?tab=repositories
* https://github.com/orgs/NightVsKnight/repositories

My LinkedIn profile is:
* https://www.linkedin.com/in/paulpv/

Since before even 2014 I have been looking for a good free open source ost/pst viewer.  
I have no idea how I never discovered this XstReader repo until today!

Before I read your news I had already envisioned updating XstReader to export to pst (and even found https://github.com/Dijji/XstReader/issues/58). I wonder if I could even make the code read .olm files from Mac Outlook (there are a few Java or Python based projects that say they can do this, but nothing in C#).

Anyways; no pressure. Wish you the best.

**Comment by seeemilyplay** [comment]
Hi @paulpv I'm so sorry about the slow reply. I've been prevaricating about this one since he died. GitHub have said they can transfer the repo to me if I furnish them with the necessary death cert etc., but I don't want this as it would involve taking the code off the original github.com/Dijji username, and I'd like to keep it there for his memory.

So I feel a bit stuck. I wish Github would think a bit harder about this sort of situation.

I'm so glad to hear that you enjoy my Dad's work, and want to be involved. He would've been delighted.

**Comment by paulpv** [comment]
@seeemilyplay Sorry if I am repeating myself with any of the below. 

My public https://github.com/paulpv GitHub profile shows my email address is pv@swooby.com.
You may contact me there if you don't want to discuss it here.

The below probably assumes GitHub can give you access to his whole account, not just this repo.

I would think you can get access to your father's GitHub account via password recovery if you have access to his email.

I think you can convert a GitHub user name (ex: `Dijji`) to an Org. Then his work could live on with other people helping. 

Alternatively, you can temporarily rename his account to something else, and then you can create a `Dijji` org, and then you can add anyone you want to the org and still be in control as long as you want.

Be careful doing this: I did something similar when I was trying to rename one of my orgs (https://github.com/OBS-NDI/OBS-NDI) to https://github.com/DistroAV/DistroAV (for "legal" reasons; OBS wants all unofficial plugins to stop using OBS in their name).
I originally just created the new org https://github.com/DistroAV , and then afterwards realized I just wanted to rename the original org. So I deleted the new org and then tried to rename the original and the GitHub UI would not let me; said the 
new name was taken.

I contacted GitHub support with an account issue and it took them a few weeks before they finally got around to my ticket and then fairly quickly fixed everything. 

I mention all of this because if you decide to create a `Dijji` org and temporarily rename `Dijji` to something else, there is a chance that GitHub won't let you create a `Dijji` org because the name is/was recently taken. If that happens, I am proof that contacting GitHub free tier support does work, it's just a little slow.

If you already have a support ticket open with them about your father's account then you may be able to tell them you want to convert his personal account to an Org with you as the admin and they may be able to make it happen.

Good luck with whatever you decide. Again (as I already said in my earlier post), I am 51 years old, have been coding for 35 years, and have 3 daughters currently 5, 7, and 9 years old, the oldest of which already likes to tinker with code. Your situation with your father struck me, and I am happy to help out however you want.

Take care.

**Comment by berkdemir** [comment]
> Before I read your news I had already envisioned updating XstReader to export to pst (and even found [#58](https://github.com/Dijji/XstReader/issues/58)). 


@paulpv can't you just fork it and start working on your own version? We can still keep his reference and link to main repo.
pst export would be a great addition to this tool.
RIP Dijji.

**Comment by raedwa01** [comment]
@seeemilyplay I know this is a very late reply, but did you ever find someone who can take over the repo?  If not, I can do it.  If you have the ability to log into your father's github, you can transfer ownership to me.  It will preserve the repository under his name, but give me the ability to approve pull requests and release updates.
